packages/components-css/icon-css/src/icon.scss

```
@@ -0,0 +1,13 @@
+@use "mixin";
+
+.nl-icon {
+  @include mixin.nl-icon;
+}
+
+.nl-icon > svg {
```

```suggestion
.nl-icon svg {
```
Gaat er iets kapot als we dit doen? Iets flexibeler